params/config.go

```diff
@@ -604,15 +613,24 @@ func (c *NodeConfig) updateClusterConfig() error {
 			return fmt.Errorf("failed to unmarshal cluster configuration file: %s", err)
 		}
 	} else {
-		clusters = defaultClusters
+		switch c.ClusterConfig.Fleet {
+		case fleetStaging:
+			clusters = stagingFleet.Cluster
+		default:
+			clusters = defaultFleet.Cluster
+		}
```

Comment:
Shouldn't this be written in such a way that we can add a fleet to the config without having to mess about with the code?

Reply:
Yes, good point. I have just changed that.